packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -17,9 +17,10 @@ import {CloudTasksClient} from '@google-cloud/tasks';
 import {v1} from '@google-cloud/secret-manager';
 import * as express from 'express';
 import pino from 'pino';
-// eslint-disable-next-line node/no-extraneous-import
 import {Octokit} from '@octokit/rest';
 import SonicBoom from 'sonic-boom';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const LoggingOctokitPlugin = require('../src/logging-octokit-plugin.js');
```

Comment:
Had to disable the `no-var-requires` here because when I changed the `const LoggingOctokitPlugin` to `import LoggingOctokitPlugin` Octokit wouldn't accept it (gave a type mismatch).

Reply:
Have you tried `import * as LoggingOctokitPlugin`

Reply:
Yeah I already tried that and the message I get is 
```
Argument of type 'typeof import("/home/asonawalla/github-repos/repo-automation-bots/packages/gcf-utils/src/logging-octokit-plugin")' is not assignable to parameter of type 'Plugin | Plugin[]'.
```
I think the issue is that the plugin is a `.js` file (as required by Octokit) and so types are messy/non-existent